README.md

```diff
@@ -78,6 +71,11 @@ python gtsfm/runner/run_scene_optimizer_colmap_loader.py --config_name {CONFIG_N
 where `COLMAP_FILES_DIRPATH` is a directory where .txt files such as `cameras.txt`, `images.txt`, etc have been saved.
 
 
+To visualize the result using Open3D, run:
+```bash
+python visualization/view_scene.py --rendering_library open3d --point_rendering_mode point
+```
```

Comment:
where does it pick up the results from? Dont see a path argument here..

Reply:
The argparse default just looks in the `results` dir, assuming we are running this inside the GTSFM repo @akshay-krishnan